package.json

```diff
@@ -10,7 +10,7 @@
   },
   "author": "Gregor Martynus (https://twitter.com/gr2m)",
   "license": "ISC",
-  "repository": "github:gr2m/squash-commit-app",
+  "repository": "https://github.com/squash-commit-app/squash-commit-app",
```

Comment:
The `github:` prefix is supported by npm

```suggestion
  "repository": "github:squash-commit-app/squash-commit-app",
```

See https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository

<img width="698" alt="image" src="https://user-images.githubusercontent.com/39992/119688071-870b3900-bdfc-11eb-8b86-32556397ff84.png">